src/js/menu/acquia_lift.personalize.collections.js

```diff
@@ -76,7 +76,7 @@
     },
 
     /**
```

Comment:
1) You are right, getVariations() was no longer referenced, so I am removing it now.
2) resetVariations() still have one use in the triggerChange() above, although removing it doesn't look like it breaks anything. I don't know how to check this for sure, though ...